docs-css/docs-sidebar.css

```diff
@@ -0,0 +1,79 @@
+.sidebar{
+    grid-area: sidebar;
+    position: sticky;
+    top: 0;
+    height: calc(100vh - 2rem);
+    overflow-y: auto;
+    min-width: 15rem;
+}
+.side-navbar{
+    display: flex;
+    flex-direction: column;
+    gap: 0.7rem;
+  }
+  .title{
+    margin-top: 1rem;
+  }
+  .subtitle{
+      color: #909396;
+      line-height: 1rem;
+  }
+  .drawer-menu{
+      height: 100%;
+      overflow-y: auto;
+  }
+  .nav-lists{
+      display: flex;
+      flex-direction: column;
+      justify-items: center;
+      gap: 0.3rem;
+      list-style: none;
+  }
+.nav-item:hover{
+    background-color: var(--secondary-light-gray);
+  }
+.nav-link .icon{
+    color: var(--dark-grey);
+    margin-right: 1.5rem;
+}
+.nav-link{
+    display: block;
+    text-decoration: none;
+    font-size: 1rem;
+    color: #3f4049;
+}
```

Comment:
Try to maintain the indentation throughout the codebase, it provides clear readability to other's who are using your code.

Reply:
Formatted the code.